src/main/java/com/github/dockerjava/netty/NettyDockerCmdExecFactory.java

```diff
@@ -580,6 +600,24 @@ public void close() throws IOException {
         eventLoopGroup.shutdownGracefully();
     }
 
+    /**
+     * Configure connection timeout in milliseconds
+     */
+    public NettyDockerCmdExecFactory withConnectTimeout(Integer connectTimeout) {
```

Comment:
Jersey has `com.github.kostyasha.yad_docker_java.com.github.dockerjava.jaxrs.JerseyDockerCmdExecFactory` so move method to DockerCmdExecFactory ?